src/ape/api/projects.py

```diff
@@ -113,10 +114,55 @@ def _create_manifest(
         if version:
             manifest.version = version
 
+        manifest.compilers = cls._create_compilers_list(source_paths, contract_types)
         manifest.sources = cls._create_source_dict(source_paths, contracts_path)
         manifest.contract_types = contract_types
+
         return manifest
 
+    @classmethod
+    def _create_compilers_list(
+        cls,
+        source_paths: List[Path],
+        contract_types: Dict[str, ContractType],
+    ):
+        compiler_list: List[Compiler] = []
+        contracts_folder = cls.config_manager.contracts_folder
+        for ext, compiler in cls.compiler_manager.registered_compilers.items():
+            sources = [x for x in source_paths if x.suffix == ext]
+            if not sources:
+                continue
+
+            try:
+                version_map = compiler.get_version_map(sources, contracts_folder)
+            except APINotImplementedError:
+                versions = list(compiler.get_versions(sources))
+                if len(versions) == 0:
+                    # Some compilers like json compiler doesn't use versioning
+                    versions = [Version("0.1.0")]
```

Comment:
@sabotagebeats Talked with @NotPeopling2day 
We think we should skip the JSON compiler for now.


```suggestion
                    continue
```

Reply:
and adjust the comment above to say we are skipping compilers that don't use versions since they are unlikely to be part of the published manifest